static/sass/_patterns_media-object--snap.scss

```diff
@@ -5,7 +5,7 @@
 
   .p-media-object--snap {
     display: block;
-    margin-bottom: $spv-inter--regular;
+    margin-bottom: $spv-inter--regular * 2;
```

Comment:
Are we sure it won't cause spacing issues in other places where this is used (like store categories)?

Reply:
As long as you set the no-margin-bottom config to the template we should be good... I imagine we'll either want no-margin (handled by the surrounding elements) or this margin (for snap wrapping). Not sure of another way to make this work without it.

Reply:
I was thinking that maybe margin should be on grid item, not here, but I'm not sure.
As for now it works, so let's leave it as it is.